spack-environment/apptainer/spack.yml

```diff
@@ -0,0 +1,11 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  specs:
+  - apptainer@1.3.1
```

Comment:
Looks like `1.3.1` is the latest version as of July 2024 (https://packages.spack.io/package.html?name=apptainer).